golem/managers/activity/mixins.py

```diff
@@ -10,6 +10,19 @@
 logger = logging.getLogger(__name__)
 
 
+class ActivityWrapperMixin:
+    def __init__(self, activity: Activity, *args, **kwargs) -> None:
+        self._activity = activity
+
+        super().__init__(*args, **kwargs)
+
+    def __getattr__(self, item):
+        result = self._activity.__getattribute__(item)
+        if callable(result):
+            return lambda *args, **kwargs: result(*args, **kwargs)
```

Comment:
Is that block is needed? Looks like wrapping callable by transparent callable.

Reply:
Seems that you are right, fixed